package.json

```diff
@@ -385,6 +385,48 @@
                     "type": "string",
                     "default": "|",
                     "description": "Separator used by statusline"
+                },
+                "vscode-neovim.compositeTimeout": {
+                    "type": "number",
+                    "default": 300,
+                    "description": "Timeout in milliseconds to wait for the next key in a composite key sequence"
+                },
+                "vscode-neovim.compositeKeys": {
+                    "type": "object",
+                    "default": {},
+                    "examples": [
+                        {
+                            "jj": {
+                                "command": "vscode-neovim.escape",
+                                "args": []
+                            }
+                        },
+                        {
+                            "jk": {
+                                "command": "vscode-neovim.lua",
+                                "args": [
+                                    "vim.api.nvim_input('<ESC>') require('vscode-neovim').action('workbench.action.files.save')"
+                                ]
+                            }
+                        }
+                    ],
+                    "patternProperties": {
+                        "^[a-zA-Z]{2}$": {
```

Comment:
@ollien VSCode does not warn about configuration errors here, not sure what's wrong.

Reply:
I'll take a look :)

Reply:
thx!

Reply:
Without altering the schema, I don't think we can do this without making the composite key itself a value (e.g. we could have an array of composite key mapping objects). There's `patternErrorMessage` in the [schema](https://code.visualstudio.com/api/references/contribution-points#Configuration-property-schema), but I don't think that works for anything but string values (i.e. not keys).

Reply:
thx! I actually forgot the vscode docs...